packages/near-snap/src/lib/utils.ts

```diff
@@ -0,0 +1,52 @@
+import type { Account } from "@near-wallet-selector/core";
+import type { GetAccountPayload } from "./types";
+
+export function hasMetaMask(): boolean {
+  if (!window.ethereum) {
+    return false;
+  }
+  return window.ethereum.isMetaMask;
+}
+
+export type GetSnapsResponse = {
+  [k: string]: {
+    permissionName?: string;
+    id?: string;
+    version?: string;
+    initialPermissions?: { [k: string]: unknown };
+  };
+};
```

Comment:
The same `GetSnapsResponse` interface exists in 2 files, can you remove this and use only the one from `types.ts`?